.gitattributes

```diff
@@ -1,15 +0,0 @@
-# Auto detect text files and perform LF normalization
```

Comment:
Just keep it removed in this PR and I'll add it back later so it doesn't clutter up the diff.